spras/domino.py

```diff
@@ -192,8 +192,10 @@ def parse_output(raw_pathway_file, standardized_pathway_file):
                     # columns that indicate edges
                     # Dropping the other rows eliminates the node information
                     module_df = pd.DataFrame(entries)
-                    module_df = module_df.loc[:, ['source', 'target']].dropna()
-
+                    try:
+                        module_df = module_df.loc[:, ['source', 'target']].dropna()
+                    except KeyError:
+                        module_df = pd.DataFrame()
```

Comment:
There are no headers for this? What is edges_df concatting on?

Reply:
I've solely removed any edges from DOMINO with "source" and "target" - all other data is preserved. 

Note that `pd.Concat` allows for empty dataframes:

```
>>> pandas.concat([pandas.DataFrame(), pandas.DataFrame()], axis=0)
Empty DataFrame
Columns: []
Index: []
```